README.md

```diff
@@ -1,14 +1,114 @@
 # Conduit Connector Oracle
 
-### General
-Oracle connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides both, a source and destination Oracle connector.
+## General
+
+Oracle connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides both, a source and
+destination Oracle connector.
+
+## Prerequisites
 
-### Prerequisites
 - [Go](https://go.dev/) 1.18
 - (optional) [golangci-lint](https://github.com/golangci/golangci-lint) 1.49.0
 
-### How to build it
+## How to build it
+
 Run `make build`.
 
-### Testing
-Run `make test`.
\ No newline at end of file
+## Testing
+
+Run `make test` to run all unit and integration tests, as well as an acceptance test. To make integration and acceptance
+tests pass, set the Oracle database URL to the environment variables as an `ORACLE_URL`.
+
+## Source
+
+The Oracle Source connects to the database using the provided `url` and starts creating records for each table row and
+each change detected.
+The first time Source runs, it creates an additional table and a trigger to track changes in the `table` and launches
+Snapshot mode. Then, when all the records have been read, Source switches to CDC mode.
+
+### Snapshot
+
+The connector in the Snapshot mode reads all rows from the table in batches via SELECT with fetching and ordering.
```

Comment:
ordering => ordering by ordering column.
And please add few words about how to important choose ordering collumn

Reply:
What do you mean? The `orderingColumn` key is required. It is very important to set it because without it the connector will not work.

Reply:
I mean it is need to chose right column . Please add this
```
OrderingColumn value must be unique and suitable for sorting, otherwise, the snapshot won't work correctly.
```

Reply:
I've added it. Line - 84.